modules/videopress/utility-functions.php

```diff
@@ -198,3 +199,59 @@ public function import( $args ) {
 	}
 	WP_CLI::add_command( 'videopress', 'VideoPress_CLI' );
 }
+
+/**
+ * Return an absolute URI for a given filename and guid on the CDN.
+ * No check is performed to ensure the guid exists or the file is present. Simple centralized string builder.
+ *
+ * @param string $guid     VideoPress identifier
+ * @param string $filename name of file associated with the guid (video file name or thumbnail file name)
+ *
+ * @return string Absolute URL of VideoPress file for the given guid.
+ */
+function videopress_cdn_file_url( $guid, $filename ) {
+	if ( is_ssl() ) {
+		return "https://videos.files.wordpress.com/{$guid}/{$filename}";
+
+	} else {
+		return "http://videos.videopress.com/{$guid}/{$filename}";
+	}
+}
+
+/**
+ * Get an array of the transcoding status for the given video post.
+ *
+ * @param int $post_id
+ * @return array|bool Returns an array of statuses if this is a VideoPress post, otherwise it returns false.
+ */
+function videopress_get_transcoding_status( $post_id ) {
+	$meta = wp_get_attachment_metadata( $post_id );
+
+	// If this has not been processed by videopress, we can skip the rest.
+	if ( !$meta || ! isset( $meta['videopress'] ) ) {
+		return false;
+	}
+
+	$info = (object) $meta['videopress'];
+
+	$status = array(
+		'std_mp4' => isset( $info->files_status ) ? $info->files_status['std']['mp4'] : null,
```

Comment:
Should this (and the subsequent three lines) be testing for the nested array value in the `isset()`?  Or maybe `! empty()`?